src/main/java/com/sourcery/gymapp/backend/workout/util/WeightComparisonUtil.java

```diff
@@ -0,0 +1,44 @@
+package com.sourcery.gymapp.backend.workout.util;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@UtilityClass
+public class WeightComparisonUtil {
+
+    private record WeightRange(int maxWeight, String message) {}
+    private static final List<WeightRange> WEIGHT_RANGES = new ArrayList<>();
+
+    static {
+        WEIGHT_RANGES.add(new WeightRange(100, "is like a baby panda! 🐼"));
```

Comment:
I guess storing this in Enum is ok 👍
But I wonder for the future: if it would be more beneficial to store it in DB.